### DISCUSSION TITLE
New demo release 11.16.0

### DISCUSSION THREAD

**Discussion by palexdev** [palexdev/MaterialFX#308] [General]
[Changelog](https://github.com/palexdev/MaterialFX/blob/main/CHANGELOG.md)

<hr /><em>This discussion was created from the release <a href='https://github.com/palexdev/MaterialFX/releases/tag/v11.16.0'>New demo release 11.16.0</a>.</em>

**Comment by RedLeafSystems** [comment]
Where is the demo file? I downloaded the `.zip` but can't find any `.java` files with the demo in them.

**Comment by infinite-dev22** [comment]
The demo Zip doesn't contain the source files rather a compiled runnable application.
For Demo source files, traverse through this [directory](https://github.com/palexdev/MaterialFX/tree/main/demo)

**Comment by Econtinent** [comment]
I got an error "jvm.dll isn't appointed to running on Windows" when I run MaterialFX Demo.bat on Win11 platform.

**Comment by elibroftw** [comment]
Slow startup on windows when running `bin/MaterialFX Demo.bat` (90 seconds!). Navigation scrolling is artificial and doesn't feel native. When switching pages, there is lag between the click and UI showing. Is this a performance bug? How is it slower than React?

**Reply by palexdev** [reply]
Latest version should have vastly improved that but I didn't release the "new" demo

**Comment by BryanGrana** [comment]
Can i add it to eclipse IDE window builder ?
